news/2022-09-27-changes-to-osu-taiko-sr-and-pp.md

```diff
@@ -0,0 +1,153 @@
+
+### Readdressing colour
+
+#### Issues within the old system
+
+Previously, the old system detected only the frequency of changes between colours and had specific penalties for particular repeated patterns. This led to complex patterns containing large quantities of single-coloured notes not being considered properly in regards to overall strain values. 
+
+In addition, a penalty was applied to repeat instances of these patterns, similar to how stamina was calculated. This only considered up to five recent occurrences, but if any two or more of such repeats occurred, a harsh penalty was applied.
+
+#### The new colour system
+
+To summarize shortly, the new colour system is more aware of *all* kinds of patterning used in osu!taiko mapping and is far more capable of formulating appropriate difficulty values from them compared to the old system.
+
+In general, the harder it is to describe a pattern, the harder the pattern will generally be considered. Additionally, the more dense a pattern (or color sequence, as described by the code) is, the harder the algorithm will consider it to be. Patterns that repeat with longer segments between them are also considered more difficult than patterns that repeat closely to one another.
+
+The code and concepts utilized to achieve this are quite extensive. If this sort of thing interests you, feel free to read more in the rework document listed below.
+
+### Calculating the final result
+
+Given the three skills briefly described above (stamina, rhythm and colour), their *strain* values have now become the number that approximates star rating. 
+
+Switching from the previous system, where strain and individual skill values formed star rating, strain is now the only summative factor in this calculation via the new *peak* skill.
+
+*Peak* difficulty splits the map into sections and combines the highest strains of each skill in each section, aiming to catch out particularly demanding moments. The final star rating is a weighted sum of the peak skill sections, including a slight multiplier to scale better.
+
+While this is all much more mathematically complex than described here, you can consult the [osu!taiko rework document](https://docs.google.com/document/d/1Z5GC4DMqOVzeIERMSK3qpQaqjq-sVnhbuoxAwy9qxDs/edit), or dive into the code over in the [osu!(lazer) GitHub repository](https://github.com/ppy/osu) for the full details.
+
+Previous approaches aimed to scale maps in regards to the [difficulty naming hierarchy](/wiki/Ranking_Criteria/Difficulty_naming), while this rework has deviated away from that goal, aiming instead to reflect true difficulty regardless of naming practices. 
+
+This also means that star rating values for maps with near-FCs are no longer near 9.5 stars, but rather can expand well into the 10 star range (medal hunters rejoice!).
```

Comment:
the what? how does star rating have anything to do with FC? confuse.

Reply:
it means that maps that tend to get many near-FCs on their leaderboards now have increased star ratings compared to before

Reply:
I think it should be described as "beatmaps which rarely see FCs achieved" if that's what it's intending to say.

Reply:
Should be worded as such then.

```suggestion
This also means that star rating values for beatmaps where top end players are almost – but not quite – getting full combo scores will now extent well into the 10 star range, rather than capping out at around 9.5 stars (medal hunters rejoice!).
```